pom.xml

```diff
@@ -3,9 +3,14 @@
      xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>net.snowflake</groupId>
+  <parent>
+    <groupId>net.snowflake</groupId>
+    <artifactId>snowflake-jdbc-base</artifactId>
+    <version>3.14.3</version>
```

Comment:
do we need to change anything in our scripts bumping versions before release?

Reply:
2 ways of managing the versions:

Option 1. The easiest way is to bump up the versions in all parent-pom.xml, pom.xml and FIPS/pom.xml so that we won't forget if there's any changes in parent-pom.xml

Option 2. For pom.xml and FIPS/pom.xml, they can have their own versions like 3.14.x and point to a specific version of parent-pom.xml by specifying `<version>` under `<parent>`. But we will need to remember to bump up version in parent-pom.xml when someone makes changes to it.

Reply:
we have internal script to bump version during release - I will create separate task for such bumping